jvb/src/main/java/org/jitsi/videobridge/stats/VideobridgeStatistics.java

```diff
@@ -314,21 +297,24 @@ private void generate0()
                         jitterSumMs += Math.abs(ssrcJitter);
                         jitterCount++;
                     }
-
                 }
 
-                PacketStreamStats.Snapshot outgoingStats
-                        = transceiverStats.getOutgoingPacketStreamStats();
-                bitrateUploadBps += outgoingStats.getBitrate();
+                PacketStreamStats.Snapshot outgoingStats = transceiverStats.getOutgoingPacketStreamStats();
+                bitrateUploadBps += outgoingStats.getBitrate().getBps();
                 packetRateUpload += outgoingStats.getPacketRate();
 
-                double endpointRtt
-                        = transceiverStats.getEndpointConnectionStats().getRtt();
+                EndpointConnectionStats.Snapshot endpointConnectionStats = transceiverStats.getEndpointConnectionStats();
+                double endpointRtt = endpointConnectionStats.getRtt();
                 if (endpointRtt > 0)
                 {
                     rttSumMs += endpointRtt;
                     rttCount++;
                 }
+
+                packetsReceivedReceived += endpointConnectionStats.getReceiveLoss().getPacketsReceived();
```

Comment:
Reading this, it feels like the endpoint connection stats method should be `getReceiveLossStats()`/`getSendLossStats()`.